Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM praekeltfoundation/django-bootstrap:py3.6
+
+COPY . /app
+
+RUN pip install -e .
```

Comment:
If you're using this image locally, a nice thing to do is to copy over all the requirements, do a pip install, and then copy over the rest of the app. That way you don't need to do a complete pip install every time you change a line of code.